x-pack/auditbeat/module/system/login/utmp.go

```diff
@@ -181,14 +181,14 @@ func (r *UtmpFileReader) readNewInFile(loginRecordC chan<- LoginRecord, errorC c
 
 	size := utmpFile.Size
 	oldSize := savedUtmpFile.Size
-	if size < oldSize {
+	if size < oldSize || utmpFile.Offset > size {
```

Comment:
not sure, but maybe we need to add this check to line 233 also

Reply:
I think you are correct, I added 2 more checks,